CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ CHANGELOG
 
 3.6.0
 ----
+**ENHANCEMENTS**
+- Add CloudWatch metrics, dashboard graphs and alarms to monitor head node memory and root volume disk utilization.
```

Comment:
What about mentioning these two metrics in the changelog: Add `mem_used_percent` and `disk_used_percent` metrics for head node memory and root volume disk utilization tracking on the ParallelCluster CloudWatch dashboard, and set up alarms for monitoring these metrics.